chia/timelord/timelord_launcher.py

```diff
@@ -16,12 +16,11 @@
 
 active_processes: List = []
 stopped = False
-lock = asyncio.Lock()
```

Comment:
Up to you on this one.

Reply:
Yeah this is causing trouble only sometimes. Creating an async lock outside of any async function causes issues, because it might be running on a different event loop. Each lock belongs to one event loop. This has caused issues in the past in various places. In this case, it causes a flaky test.

We might be able to remove the lock, but i'm not 100% sure right now. If we do, I would prefer to do it in a separate PR.